src/test/java/GudePDU8045CommunicatorTest.java

```diff
@@ -449,4 +451,25 @@ void testDynamicStatistic() throws Exception {
 		Assertions.assertNotNull(dynamicStats.get("SensorPort02#Temperature(C)"));
 	}
 
+	@Test
```

Comment:
Please add javadocs with descriptions and expectation

Reply:
I fixed it